de.dlr.sc.virsat.external.lib.jersey/pom.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+
+This program and the accompanying materials are made available under the
+terms of the Eclipse Public License 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.
+
+SPDX-License-Identifier: EPL-2.0
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>de.dlr.sc.virsat.parent</artifactId>
+		<groupId>de.dlr.sc.virsat</groupId>
+		<version>4.10.0-SNAPSHOT</version>
+		<relativePath>../maven/pom.xml</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>de.dlr.sc.virsat.external.lib.jersey</artifactId>
+	<name>de.dlr.sc.virsat.external.lib.jersey</name>
+	<packaging>eclipse-plugin</packaging>
+
+
+</project>
```

Comment:
I think not having file endings somethimes couses problems on linux... Could we fix that?

Reply:
Not sure about linux, but i don't mind adding it.